tools/watchnrtattr/watchnrtattr.go

```diff
@@ -59,7 +60,8 @@ func main() {
 		nrtObjs := topologyv1alpha2.NodeResourceTopologyList{}
 		wa, err := cli.Watch(ctx, &nrtObjs)
 		if err != nil {
-			log.Fatalf("cannot watch NRT objects: %v", err)
+			klog.Errorf("cannot watch NRT objects: %v", err)
+			return
```

Comment:
why this one?

Reply:
same as above, in case of fatal error the program will exit without executing deferred code